payment/payment-repository/src/main/java/shop/woowasap/payment/repository/entity/PaymentEntity.java

```diff
@@ -0,0 +1,54 @@
+package shop.woowasap.payment.repository.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+import java.math.BigInteger;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import shop.woowasap.payment.domain.PayStatus;
+import shop.woowasap.payment.domain.PayType;
+
+@Entity
+@Getter
+@NoArgsConstructor
+@Table(name = "payment", indexes = {@Index(name = "index_order_id", columnList = "order_id")})
+public class PaymentEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "payment_id", nullable = false)
+    private Long paymentId;
+
+    @Column(name = "user_id", nullable = false)
+    private Long userId;
+
+    @Column(name = "order_id", nullable = false)
+    private Long orderId;
```

Comment:
orderId와 paymentId가 따로 관리되어있고, orderId에 인덱스가 걸려있으면, orderId기반으로 payment를 찾을때, 인덱스를 2번 검색하겠네요!

orderId로 PaymentId위치 찾은다음에, paymentId로 인덱스 찾는방식?

```suggestion
    @Id
    @Column(name = "payment_id", nullable = false)
    private Long paymentId;

    @Column(name = "user_id", nullable = false)
    private Long userId;

    @Column(name = "order_id", nullable = false)
    private Long orderId;
```